tool/tsh/app.go

```diff
@@ -106,13 +106,14 @@ func onAppLogin(cf *CLIConf) error {
 			"appName": app.GetName(),
 		})
 	}
-	curlCmd, err := formatAppConfig(tc, profile, app.GetName(), app.GetPublicAddr(), appFormatCURL, rootCluster)
+	curlCmd, err := formatAppConfig(tc, profile, app.GetName(), app.GetPublicAddr(), appFormatCURL, rootCluster, cf.InsecureSkipVerify)
```

Comment:
Can you get the "insecure" flag from Telelport Client (`tc` variable) instead of passing yet another flag?